iac-extensions/kubernetes/src/test/java/org/sonar/iac/helm/tree/utils/GoTemplateAstHelperTest.java

```diff
@@ -92,4 +94,22 @@ void shouldFindValuesForTwoHelmExpressionsInRange() throws IOException {
 
     assertThat(actual).contains(new ValuePath("Values", "hostIPC"), new ValuePath("Values", "foo", "bar"));
   }
+
+  @ParameterizedTest
+  @ValueSource(strings = {
+    " ", "​", "©"
+  })
+  void shouldNotThrowWithSpecialSymbols(String specialSymbol) {
+    var code = """
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: example
+      spec:
+        data:
+         \s""" + specialSymbol;
+
+    assertThatCode(() -> new GoAstCreator(tempDir).goAstFromSource(code, "", "name: test chart"))
```

Comment:
Did it throw when you didn't have your fix in place?
If yes good, if no I would like an example for that :D

Reply:
Yes, it did :)